File: VB/HowToExdendApplicationModelAndSchema.Module.Win/ModelExtender.vb
Imports System
Imports System.ComponentModel
Imports DevExpress.Data

Public Interface IModelListViewExtender
    Property IsGroupFooterVisible() As Boolean
End Interface

Public Interface IModelColumnExtender
    <DefaultValue(SummaryItemType.None)> Property GroupFooterSummaryType() As SummaryItemType
End Interface

File: CS/HowToExdendApplicationModelAndSchema.Module.Win/ModelExtender.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.Data;
using DevExpress.ExpressApp.DC;
using System.ComponentModel;

namespace HowToExdendApplicationModelAndSchema.Module.Win {
    public interface IModelListViewExtender {
        bool IsGroupFooterVisible { get;set;}
    }

    public interface IModelColumnExtender {
        [DefaultValue(SummaryItemType.None)]
        SummaryItemType GroupFooterSummaryType { get;set;}
    }
}
